config/runtime.exs

```diff
@@ -6,14 +6,6 @@ config :screenplay, ScreenplayWeb.Endpoint,
   url: [host: System.get_env("HOST"), port: 80],
   secret_key_base: System.get_env("SECRET_KEY_BASE")
 
-sftp_client_module =
-  case System.get_env("SFTP_SERVER") do
-    "outfront" -> SFTPClient
-    _ -> Screenplay.Outfront.FakeSFTPClient
-  end
-
-env = System.get_env("ENVIRONMENT_NAME")
-
```

Comment:
The changes in this file now appear to be a no-op — just moving things around. Can we revert it back to the original content?

Reply:
I did make a small change to the `case` so it looks at `env` now. There was no reason I should have moved the code to a different part of the file. Moving it back to where it was.